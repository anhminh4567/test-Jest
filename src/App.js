import Navigation from "./components/Navigation";
import "./App.css";
import Player from "./components/Players";
import PlayerMap from "./components/PlayerMap";
import Main from "./components/Main";
import Product  from "./Products/Product";

const list = ['abc','def','ghj','klm'];

function App() {
  return (
    <div className='App'>
      <Navigation />
      {/* {<Player /> }
      {<PlayerMap /> } */}
      <Main />
    </div>
  );
  // return (
  //   <div className="App">
  //     <header className="App-header">
  //       <h2>
  //         React snapshot test
  //       </h2>
  //       <Product items={list}> 

  //       </Product>
  //       <h3>
  //         the above will change
  //       </h3>
  //     </header>
  //   </div>
  // );
}

export default App;
