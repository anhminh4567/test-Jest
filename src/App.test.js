import { render, screen } from "@testing-library/react";
import App from "./App";
import renderer from "react-test-renderer"
// test("renders learn react link", () => {
//   render(<App />);
//   const linkElement = screen.getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });
test("render snapshot when there are no parameter",() => {
  const tree =renderer.create(<App></App>).toJSON();
    expect(tree).toMatchSnapshot();
});
