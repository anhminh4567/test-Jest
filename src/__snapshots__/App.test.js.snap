// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render snapshot when there are no parameter 1`] = `
<div
  className="App"
>
  <div>
    <nav>
      <ul>
        <li>
          <a
            className="active"
            href="#home"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="#news"
          >
            News
          </a>
        </li>
        <li>
          <a
            href="#about"
          >
            About
          </a>
        </li>
        <li>
          <a
            href="#contact "
          >
            Contact
          </a>
        </li>
      </ul>
    </nav>
  </div>
  <div
    className="container"
  >
    <div
      className="column"
    >
      <div
        className="card"
      >
        <img
          alt=""
          src="assets/images/cr.jpg"
        />
        <h3>
          Cristiano Ronaldo
        </h3>
        <p
          className="title"
        >
          Manchester United
        </p>
        <p>
          <button>
            Detail
          </button>
        </p>
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="card"
      >
        <img
          alt=""
          src="assets/images/kante.jpg"
        />
        <h3>
          Kante
        </h3>
        <p
          className="title"
        >
          Chelsea
        </p>
        <p>
          <button>
            Detail
          </button>
        </p>
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="card"
      >
        <img
          alt=""
          src="assets/images/messi.jpg"
        />
        <h3>
          Messi
        </h3>
        <p
          className="title"
        >
          PSG
        </p>
        <p>
          <button>
            Detail
          </button>
        </p>
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="card"
      >
        <img
          alt=""
          src="assets/images/neymar.jpg"
        />
        <h3>
          Neymar
        </h3>
        <p
          className="title"
        >
          PSG
        </p>
        <p>
          <button>
            Detail
          </button>
        </p>
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="card"
      >
        <img
          alt=""
          src="assets/images/kane.jpg"
        />
        <h3>
          Kane
        </h3>
        <p
          className="title"
        >
          Tottemham
        </p>
        <p>
          <button>
            Detail
          </button>
        </p>
      </div>
    </div>
    <div
      className="column"
    >
      <div
        className="card"
      >
        <img
          alt=""
          src="assets/images/haaland.jpg"
        />
        <h3>
          Haaland
        </h3>
        <p
          className="title"
        >
          Manchester City
        </p>
        <p>
          <button>
            Detail
          </button>
        </p>
      </div>
    </div>
  </div>
</div>
`;
